.github/workflows/develop.yml

```diff
@@ -141,6 +141,7 @@ jobs:
           RISCV=`pwd`/../riscv ./test.sh --build-only
           cd ..
       - name: Run test suite
+        # It is also possible to use arm64v8/rust@sha256:cd7c932094e80f2f2471af1c2c8dba4e87221bc55318ee846d003f2f4e26970a
```

Comment:
We can remove this here? The reason I want to stick to `latest` tag in docker image, is that CI should be able to pick up docker image upgrades automatically, so we don't miss problems due to sticking to an older version